docs/source/installation.rst

```diff
@@ -600,70 +573,61 @@ The only thing to note is that the GPU driver you are currently using should be
-   set JAVA_HOME=%ProgramFiles%\Java\graalvm-jdk-21.0.1+12.1
-   
-   rem SUBSTITUTE <TornadoVM> WITH ACTUAL DIRECTORY
-   set PATH=<TornadoVM>\bin\bin;<TornadoVM>\level-zero\build\bin\Release;%PATH%
+If you haven't configured Visual Studio 2022 to use C++, you may need to install it using the Visual Studio Installer. 
+In this case, enable the following packages:
 
-   rem SUBSTITUTE <TornadoVM> WITH ACTUAL DIRECTORY
-   set TORNADO_SDK=<TornadoVM>\bin\sdk
+- MSVC C++ x86/64 build tools (latest)
+- MSVC C++ x86/64 Spectre-mitigated libs (latest)
+- C++ ATL for latest build tools (latest for x86/64)
+- C++ ATL for latest build tools with Spectre Mitigations (x86/64)
 
-   rem NEXT LINE IS NECESSARY TO BUILD PTX (NVIDIA CUDA) BACKEND (USUALLY SET BY CUDA INSTALLER)
-   set CUDA_PATH=C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.1
 
-Check each variable to meet your configuration. Especially substitute ``<TornadoVM>`` in ``PATH`` and ``TORNADO_SDK`` with your TornadoVM directory.
-
-6. Compile TornadoVM
-~~~~~~~~~~~~~~~~~~~~~~~
+4. Download TornadoVM
+~~~~~~~~~~~~~~~~~~~~~~~~
 
-Start a ``cmd.exe`` terminal, navigate to the ``<TornadoVM>`` directory, and build TornadoVM as follows:
+Clone the latest TornadoVM source code from the GitHub `repository <https://github.com/beehive-lab/TornadoVM>`__:
 
 .. code:: bash
 
-   cd %USERPROFILE%\MyProjects\TornadoVM
-   etc/setvars.cmd
-   nmake /f Makefile.mak graal-jdk-21 BACKEND=opencl,ptx
+   cd %USERPROFILE%\MyProjects
+   git clone https://github.com/beehive-lab/TornadoVM.git
+   cd TornadoVM
 
-The ``BACKEND`` parameter has to be a comma-separated list of ``ptx``, ``spirv`` and ``opencl`` options. You may build ``ptx`` only when NVIDIA GPU Computing Toolkit (CUDA) is installed. The ``spirv`` backend makes use of the `Intel Level Zero API <https://dgpu-docs.intel.com/technologies/level-zero.html>`__.
+We will refer hereafter the directory with TornadoVM sources as ``<TornadoVM>``.
 
-7. Check the installation
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Don't close ``cmd.exe`` after the build. Run the following command to see that TornadoVM is working:
+5. Configure/Compile the TornadoVM Project 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 .. code:: bash
 
-   python %TORNADO_SDK%\bin\tornado --devices
-
-You should see a list of OpenCL and/or CUDA devices available on your system.
-
-Now try to run a simple test. To run examples with Graal JDK 21, TornadoVM uses modules:
-
-.. code:: bash
+   python -m venv .venv
+   .venv\Scripts\activate.bat
+   .\bin\windowsMicrosoftStudioTools2022.cmd
+   python bin\tornadovm-installer --jdk jdk21 --backend=opencl 
+   setvars.cmd
 
-   python %TORNADO_SDK%\bin\tornado -m tornado.examples/uk.ac.manchester.tornado.examples.compute.MatrixMultiplication2D --params="512"
 
-To run individual tests:
+6. Check the installation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 .. code:: bash
+   # explore accelerator reachable from TornadoVM
```

Comment:
```suggestion

   # list the accelerator devices that are reachable from TornadoVM
```

Reply:
I added a space line in line 622, because the code snippet (beneath it) is not shown.